tools/flaky-test-retryer/github_commenter.go

```diff
@@ -128,29 +188,14 @@ func parseEntries(comment *github.IssueComment) (map[string]*entry, error) {
 	if comment == nil {
 		return entries, nil
 	}
-	re := regexp.MustCompile(`.* \| \d`)
-	entryStrings := re.FindAll([]byte(comment.GetBody()), -1)
+	re := regexp.MustCompile(`.* \| \d/\d`)
+	entryStrings := re.FindAllString(comment.GetBody(), -1)
```

Comment:
Maybe take a string arg and pass the body() when calling this?

Reply:
sorry but I'm not quite get what you meant

Reply:
we dont need to take the comment struct as an arg. We only need to parse the body and can just accept a string as arg instead.

Reply:
agree, done